cvmfs/session_context.cc

```diff
@@ -337,7 +342,6 @@ bool SessionContext::DoUpload(const SessionContext::UploadJob* job) {
   curl_easy_setopt(h_curl, CURLOPT_USERAGENT, "cvmfs/" VERSION);
   curl_easy_setopt(h_curl, CURLOPT_MAXREDIRS, 50L);
   curl_easy_setopt(h_curl, CURLOPT_CUSTOMREQUEST, "POST");
-  curl_easy_setopt(h_curl, CURLOPT_TCP_KEEPALIVE, 1L);
```

Comment:
That's interesting!  I have to remember to ask you what the effect of this change was to the system.

Reply:
This has no effect visible effect in the end, I just wanted to go back to the default. The problem was coming from the Erlang Cowboy server, who had a timeout by default which was making my requests fail.

Now, about CURLOPT_TCP_KEEPALIVE, this is a bit misleading, in my opinion. This only controls sending TCP keepalive probes not the connection keepalive mechanism. The default is 0 (disabled) which is what I want.